.github/CODEOWNERS

```diff
@@ -28,6 +28,7 @@
 /gnatsd/                @stephenprater @jaredhoyt dev@goldstar.com
 /gnatsd_streaming/      @stephenprater @jaredhoyt dev@goldstar.com
 /gremlin/               support@gremlin.com
+/hasura_cloud/          support@hasura.io
```

Comment:
There needs to be a github username or team handle before an email here. Validation is failing with the following error:
`Integration hasura_cloud has a `CODEOWNERS` entry, but the codeowner is not a username or team.`